### ISSUE TITLE
Add support for ignoring object names via patterns

### ISSUE THREAD

**Issue by lavelle** [wix/eslint-plugin-lodash#88]
Would it be possible to add this feature to the rule `prefer-lodash-methods`?

Rather than having the array of `ignoreObjects` giving exact object names, it could be an array of patterns to match. In our codebase we have a naming convention of all jQuery objects starting with a `$` symbol, and we want to filter these out so methods like `jQuery.find` and `jQuery.filter` are not flagged as false positives.

The patterns could be a regex, so in our case

```
"ignorePatterns": ["^\$[a-zA-Z0-9\_]+"]
```

to match all object names starting with a dollar symbol.

Do you think this would be a good API for this? Happy to submit a pull request if you think it would be useful.

**Comment by lavelle** [comment]
i just went ahead and made a pr for this

**Comment by lavelle** [comment]
bump, any chance you could take a look at my PR
